### PR TITLE
Update for field binding

### PR DESCRIPTION
**Pull request by albertjan**
updates the `BodyDeserializer` implementation in to make use of the new `MemberBindingInfo`

### CODE REVIEW

**Comment by PureKrome** [comment]
@albertjan this PR updates Json to 6.0.6 ? Which would make #16 redundant?

**Comment by albertjan** [comment]
@PureKrome it does

File: src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs [hunk 1]
```diff
@@ -52,8 +52,16 @@ public object Deserialize(string contentType, Stream bodyStream, BindingContext
...
+            var properties = 
+                context.DestinationType.GetProperties(BindingFlags.Public | BindingFlags.Instance)
+                    .Select(p => new BindingMemberInfo(p));
             
-            if (context.DestinationType.GetProperties(BindingFlags.Public | BindingFlags.Instance).Except(context.ValidModelProperties).Any())
+            var fields = 
+                context.DestinationType.GetFields(BindingFlags.Public | BindingFlags.Instance)
+                    .Select(f => new BindingMemberInfo(f));
+
+            if (properties.Concat(fields).Except(context.ValidModelBindingMembers).Any())
```
Comment (khellang) [inline src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs]: Does this work? Has the `Equals` method been overridden on the `BindingMemberInfo` type?

Comment (albertjan) [inline src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs]: Good one. It isn't implemented, and therefore it probably doesn't work.

Comment (albertjan) [inline src/Nancy.Serialization.JsonNet/JsonNetBodyDeserializer.cs]: Preparing a PR.

**Comment by khellang** [comment]
:ok_hand:

**Pull request merged**